CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed bugs
 
+ * Fixed panic when parsing conflict markers of a particular form.
```

Comment:
I'd add the word "invalid" before conflict markers

Reply:
Done